Fix: correct typo and "Hashcat" to "hashcat"

DhruvTheDev1 opened the pull request:
- Fixed typo: RUN_EXTENTED to  RUN_EXTENDED
- Capitalised Hashcat

DhruvTheDev1 committed:
Update hashcat-assimilation-bridge-development.md

jsteube commented:
It’s part of our style, we intentionally write hashcat in lowercase. If you see it capitalized, that’s a typo. So the basic rule is: always use lowercase hashcat.

DhruvTheDev1 committed:
Update hashcat-assimilation-bridge-development.md

DhruvTheDev1 commented:
> It’s part of our style, we intentionally write hashcat in lowercase. If you see it capitalized, that’s a typo. So the basic rule is: always use lowercase hashcat.

Hi, Thanks for the feedback!!
I changed all the "Hashcat" to "hashcat"
Also fixed the typo RUN_EXTENTED to RUN_EXTENDED
